### PR TITLE
[ci_nmstate] ci_nmstate OCP 4.15 fixes

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Review by cescgina — APPROVED** [review]
/approve

File: roles/ci_nmstate/molecule/default/converge.yml [hunk 1]
```diff
@@ -29,7 +31,12 @@
         name: crc
         ansible_ssh_private_key_file: "{{ ansible_user_dir }}/.crc/machines/crc/id_ecdsa"
         ansible_ssh_user: core
+        cifmw_molecule_ci_nmstate_crc_mac: "{{ cifmw_molecule_ci_nmstate_crc_mac }}"
 
+    - name: Add a dummy ethernet interface to target the crc NMstate
```
Comment (cjeanner) [inline roles/ci_nmstate/molecule/default/converge.yml:36]: I'd do that while CRC is still stopped - iirc we start it from within the "prepare.yml" ?
There might be some issues with the amount of available PCI ports or something, with "hot-plugging" the NIC.

Comment (cjeanner) [reply roles/ci_nmstate/molecule/default/converge.yml:37]: OK with comment to warn about the PCI thingy.

**Pull request merged**